komga-webui/src/router.ts

```diff
@@ -1,5 +1,5 @@
 import Vue from 'vue'
-import Router from 'vue-router'
+import Router, { Route } from 'vue-router'
```

Comment:
why do you need this import, it doesn't seem to be used in the file?

Reply:
Mentioned in other comment